frontend/translations/es.po

```diff
@@ -5993,6 +5993,21 @@ msgstr "Copiar"
 msgid "workspace.shape.menu.copy_link"
 msgstr "Copiar enlace al portapapeles"
 
+msgid "workspace.shape.menu.copy_paste_as"
+msgstr "Copiar/Pegar como ..."
+
+msgid "workspace.shape.menu.copy_css"
+msgstr "Copiar como CSS"
+
+msgid "workspace.shape.menu.copy_css_nested"
```

Comment:
Prefer using `-` unless the `_` is strictly necessary.

Reply:
ok